spec/models/user_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject(:user) do
+    User.new(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.',
+             posts_counter: 0)
```

Comment:
Good job on your test coverage! However, I noticed that while we have tests for negative and zero values for `posts_counter`, we don't have an explicit test for positive values. To ensure comprehensive coverage, consider adding a test case like:

```ruby
it 'is valid with a positive posts_counter value' do
    user.posts_counter = 1
    expect(user).to be_valid
end

Reply:
Thank you very much sir for the observation

Reply:
@flemton you're welcome